.github/workflows/pr-main_l1.yaml

```diff
@@ -150,34 +150,57 @@ jobs:
             ethrex_flags: ""
+            hive_flags: ""
           - name: "Engine Auth and EC tests"
             simulation: ethereum/engine
             test_pattern: engine-(auth|exchange-capabilities)/
             ethrex_flags: ""
+            hive_flags: ""
           - name: "Cancun Engine tests"
             simulation: ethereum/engine
             test_pattern: "engine-cancun/Blob Transactions On Block 1|Blob Transaction Ordering, Single|Blob Transaction Ordering, Multiple Accounts|Replace Blob Transactions|Parallel Blob Transactions|ForkchoiceUpdatedV3|ForkchoiceUpdatedV2|ForkchoiceUpdated Version|GetPayload|NewPayloadV3 After Cancun|NewPayloadV3 Before Cancun|NewPayloadV3 Versioned Hashes|Incorrect BlobGasUsed|ParentHash equals BlockHash|RPC:|in ForkchoiceState|Unknown SafeBlockHash|Unknown FinalizedBlockHash|Unique|Re-Execute Payload|Multiple New Payloads|NewPayload with|Build Payload with|Re-org to Previously|Safe Re-Org to Side Chain|Transaction Re-Org|Re-Org Back into Canonical Chain, Depth=5|Suggested Fee Recipient Test|PrevRandao Opcode|Fork ID: *|Invalid Missing Ancestor Syncing ReOrg|Request Blob Pooled Transactions"
             ethrex_flags: ""
+            hive_flags: ""
           - name: "Paris Engine tests"
             simulation: ethereum/engine
             test_pattern: "engine-api/RPC|Re-org to Previously Validated Sidechain Payload|Re-Org Back into Canonical Chain, Depth=5|Safe Re-Org|Transaction Re-Org|Inconsistent|Suggested Fee|PrevRandao Opcode Transactions|Fork ID|Unknown SafeBlockHash|Unknown FinalizedBlockHash|Unique Payload ID|Re-Execute Payload|Multiple New Payloads|NewPayload with|Payload Build|Build Payload"
             ethrex_flags: ""
+            hive_flags: ""
           - name: "Engine withdrawal tests"
             simulation: ethereum/engine
             test_pattern: "engine-withdrawals/engine-withdrawals test loader|GetPayloadV2 Block Value|Sync after 2 blocks - Withdrawals on Genesis|Max Initcode Size|Pre-Merge Fork Number > 0|Empty Withdrawals|Corrupted Block Hash Payload|Withdrawals Fork on Block 2|Withdrawals Fork on Block 3|GetPayloadBodies|Withdrawals Fork on Block 1 - 1 Block Re-Org|Withdrawals Fork on Block 1 - 8 Block Re-Org NewPayload|Withdrawals Fork on Block 8 - 10 Block Re-Org NewPayload|Withdrawals Fork on Canonical Block 8 / Side Block 7 - 10 Block Re-Org [^S]|Withdrawals Fork on Canonical Block 8 / Side Block 9 - 10 Block Re-Org [^S]"
             ethrex_flags: ""
+            hive_flags: ""
           - name: "Sync full"
             simulation: ethereum/sync
             test_pattern: ""
             ethrex_flags: "--syncmode full"
+            hive_flags: ""
           - name: "Sync snap"
             simulation: ethereum/sync
             test_pattern: ""
             ethrex_flags: "--syncmode snap"
+            hive_flags: ""
+          - name: "Eest Consume-Engine (Prague)"
```

Comment:
Lets see how long these take to see if we add it to every PR or just for the daily reports (since we already have blockchain tests that in theory are the same set of tests (but without using the engine api).

Reply:
They take quite a lot. I would just run it daily (daily_reports), both with revm and levm. Thoughts?

Reply:
Prague and Shanghai are still under Assertor Stability Check time so it shouldn't be an issue (Cancún is being cancelled due to failures, but it shouldn't take much longer)

Reply:
The problem is also that I guess we will exceed the docker quota more often since we're adding more jobs